platform/lib/format-transform/transforms.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const config = require('@lib/config.js');
+
+const host = config.getHost(config.hosts.platform);
+
+function absoluteUrl(url) {
+  if (!url.match(/^https?:\/\//)) {
+    url = host + url;
+  }
+  return url;
+}
+
+const amp4emailTransform = {
+  'html': (el) => {
+    el.removeAttr('amp');
+    el.removeAttr('⚡');
+    el.attr('⚡4email', '');
+  },
+  'form, a': (el) => {
+    el.removeAttr('target');
+  },
+  'amp-list': (el) => {
+    el.removeAttr('reset-on-refresh');
```

Comment:
These transformations are dangerous as they break functionality. Instead, I'd throw an exception and mark the file as not transformable.